user/kyotocabinet/template.py

```diff
@@ -0,0 +1,14 @@
+pkgname = "kyotocabinet"
+pkgver = "1.2.80"
+pkgrel = 0
+build_style = "configure"
+configure_args = ["--prefix=/usr"]
```

Comment:
```suggestion
build_style = "gnu_configure"
```

Reply:
Much thanks! It's my first time contributing on cports, so there might be some template issues since I copied those settings from another template, again thanks for pointing out. Should have autosplit enabled. Next time will commit one template at one time as well. Plus one question: What's the main difference between configure and gnu_configure? I managed to build libpinyin successfully on my Chimera environment. Not sure about the pros and cons between them.

Reply:
> What's the main difference between configure and gnu_configure?
gnu_configure is for autotools and this project is autotools, see the description of configure/gnu_configure in packaging.md

> Next time will commit one template at one time as well.

you did this already, and it's correct, the only thing that is incorrect is the commit message.

Reply:
by default these two packages installs its files into /usr/local, and that's referred by cbuild as a forbidden path, so I had to put "--prefix=/usr" there. (ran it on cbuild, same behavior) And kyotocabinet supplied its pacakge without configure.ac etc. / with prebuilt configure file. so no need to change build_style for it.

Reply:
> by default these two packages installs its files into /usr/local,

autotools defaults to --prefix=/usr/local. and `gnu_configure` passes `--prefix=/usr` for you.

the configure.in (.ac is not the only thing that is autotools) that kyotocabinet comes with is a gnu_configure script, so this is supposed to be gnu_configure